source/manual/2nd-line.html.md

```diff
@@ -7,14 +7,22 @@ section: 2nd line
 type: learn
 ---
 
-2nd line has three main responsibilities:
+2ndline is the user support function of GOV.UK. 2nd line’s main responsibilities:
 
 - Monitoring the state of the GOV.UK infrastructure
 - Investigating and responding to technical bug reports
 - Providing first line support to queries from data.gov.uk users
```

Comment:
I wonder if this needs updating too? About a year ago (IIRC) we made a switch whereby User Support handle most DGU queries in the first instance. We then provide 2nd line support for the queries they're unable to resolve.

Reply:
ok, good spot. I think then rather than including that somewhat muddled definition of our relationship to DGU, we remove that line and it falls cleanly enough in the line underneath: 'Taking on urgent work or work that doesn’t necessarily belong in any team'. 

what do you think?

Reply:
That would be fine 👍